src/compiler/transformers/transform-utils.ts

```diff
@@ -23,7 +23,27 @@ export const isMemberPrivate = (member: ts.ClassElement): boolean => {
   return false;
 };
 
-export const convertValueToLiteral = (val: any, refs: WeakSet<any> = null) => {
+/**
+ * Convert a JavaScript value to the TypeScript internal representation for a
+ * literal AST node with that same value. The value to convert may be a
+ * primitive type like `string`, `boolean`, etc or may be an `Object`, `Array`,
+ * etc.
+ *
+ * Note that this function takes a param (`refs`) with a default value,
+ * normally a value should _not_ be passed for this parameter since it is
+ * intended to be used for recursive calls.
+ *
+ * @param val the value to convert
+ * @param refs a set of references, used in recursive calls to avoid
+ * circular references when creating object literal IR instances. **note that
```

Comment:
Same thing for this file - can we expand the first usage of 'IR' for each JSDoc?